packages/dds/merge-tree/src/collections/rbTree.ts

```diff
@@ -637,26 +637,23 @@ export class RedBlackTree<TKey, TData> implements SortedDictionary<TKey, TData>
 		let go = true;
 		if (node) {
 			if (actions.pre) {
-				// eslint-disable-next-line @typescript-eslint/prefer-nullish-coalescing -- ?? is not logically equivalent when the first clause returns false.
-				if (actions.showStructure || node.color === RBColor.BLACK) {
+				if (!!actions.showStructure || node.color === RBColor.BLACK) {
```

Comment:
I find the ignore here and reasoning to be confusing/not particularly pretty. Replaced with `!!` to explicitly convert the condition from null/undefined to a boolean. Also open to keeping the ignore or changing how we coerce things to booleans. In my experience `!!` has been the most idiomatic, but there are a number of other ways to achieve the same effect

Reply:
seems fine